UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/UI_Ai.cs

```diff
@@ -16,56 +16,43 @@ namespace UI.Systems.MainHUD.UI_Bottom
 {
 	public class UI_Ai : MonoBehaviour
 	{
-		[HideInInspector]
-		public AiPlayer aiPlayer = null;
+		[HideInInspector] public AiPlayer aiPlayer = null;
 
-		[HideInInspector]
-		public AiMouseInputController controller = null;
+		[HideInInspector] public AiMouseInputController controller = null;
 
 		//Laws Tab Stuff
-		[SerializeField]
-		private GameObject aiLawsTab = null;
+		[SerializeField] private GameObject aiLawsTab = null;
 
-		[SerializeField]
-		private Transform aiLawsTabContents = null;
+		[SerializeField] private Transform aiLawsTabContents = null;
 
-		[SerializeField]
-		private GameObject aiLawsTabDummyLaw = null;
+		[SerializeField] private GameObject aiLawsTabDummyLaw = null;
 
-		[SerializeField]
-		private TMP_Text amountOfLawsText = null;
+		[SerializeField] private TMP_Text amountOfLawsText = null;
 
 		//Slider Stuff
-		[SerializeField]
-		private Slider powerSlider = null;
+		[SerializeField] private Slider powerSlider = null;
 
-		[SerializeField]
-		private Slider integritySlider = null;
+		[SerializeField] private Slider integritySlider = null;
 
 		//Call Shuttle Stuff
-		[SerializeField]
-		private GameObject callShuttleTab = null;
+		[SerializeField] private GameObject callShuttleTab = null;
 
 		[SerializeField]
 		private TMP_InputField callReasonInputField = null;
 
 		//Camera Teleport Screen
-		[SerializeField]
-		private TeleportWindow teleportWindow = null;
+		[SerializeField] private TeleportWindow teleportWindow = null;
 
 		//Camera Save
-		[SerializeField]
-		private GameObject cameraSaveDummy = null;
+		[SerializeField] private GameObject cameraSaveDummy = null;
 
-		[SerializeField]
-		private Transform cameraSaveContents = null;
+		[SerializeField] private Transform cameraSaveContents = null;
 
 		//First is the UI button, second is the associated security camera object
 		private Dictionary<GameObject, GameObject> savedCameras = new Dictionary<GameObject, GameObject>();
 
 		//Number of cameras
-		[SerializeField]
-		private TMP_Text numberOfCameras = null;
+		[SerializeField] private TMP_Text numberOfCameras = null;
```

Comment:
Uh, looks so much better and readable when the attribute is on top

Reply:
Might be whatever IDE is being used.

Reply:
Rider and I would disagree

Reply:
I prefer two lines too, because if you have enough attributes the line becomes too long, prompting a new-line for those, leading to inconsistency. But eh. Not worth enforcing I guess.

Reply:
Since all variables in the script has one attribute I don't see why they should be decoupled above them. If there were multiple variables that have more than one attribute then sure, it would make sense to put them on top of them.
But hey, there's no guidelines against that so ¯\_(ツ)_/¯

Reply:
Also you havent changed anything in this file, so can you revert it